src/public/settings.ts

```diff
@@ -10,8 +11,11 @@ import { getGenericOnCompleteHandler } from '../internal/utils';
 export namespace settings {
   let saveHandler: (evt: SaveEvent) => void;
   let removeHandler: (evt: RemoveEvent) => void;
-  GlobalVars.handlers['settings.save'] = handleSave;
-  GlobalVars.handlers['settings.remove'] = handleRemove;
+
+  export function initialize(): void {
+    registerHandler('settings.save', handleSave, false);
+    registerHandler('settings.remove', handleRemove, false);
```

Comment:
why not do it inside this function registerOnRemoveHandler?

Reply:
Because they both have default behavior, I kept the old pattern for when there is more work to do than just calling a handler (see also Handlers for more examples)